tests/h/search/query_test.py

```diff
@@ -1,12 +1,48 @@
 # -*- coding: utf-8 -*-
 from __future__ import unicode_literals
 
+import datetime
+
 import pytest
 import webob
 
 from h.search import Search, index, query
 
 
+class TestBuilder(object):
+    @pytest.mark.parametrize("sort_key,order,expected_order", [
+        # Sort supports "updated" and "created" fields.
+        ("updated", "desc", [0, 1, 2]),
+        ("updated", "asc", [2, 1, 0]),
+        ("created", "desc", [2, 1, 0]),
+        ("created", "asc", [0, 1, 2]),
+
+        # Default sort order should be descending.
+        ("updated", None, [0, 1, 2]),
+
+        # Default sort field should be "updated".
+        (None, "asc", [2, 1, 0]),
+    ])
+    def test_it_sorts_annotations(self, Annotation, search, sort_key, order, expected_order):
+        dt = datetime.datetime
+        ann_ids = [Annotation(updated=dt(2018, 1, 1), created=dt(2016, 1, 1)).id,
+                   Annotation(updated=dt(2017, 1, 1), created=dt(2017, 1, 1)).id,
```

Comment:
This is a bit nit picky, but these 3 annotations are in an order(desc by updated and asc by created). Could we shuffle them a bit, maybe just switch the first 2 annotations? Otherwise LGTM

Reply:
Done. I put the `2017` annotation first so that creation order is different to sorted order in all cases and added a comment to explain the creation ordering.